src/Microsoft.Dnx.DesignTimeHost/ApplicationContext.cs

```diff
@@ -1097,9 +1100,9 @@ private State DoInitialWork(string appPath, string configuration, bool triggerBu
             return state;
         }
 
-        private ApplicationHostContext GetApplicationHostContext(Project project, FrameworkName frameworkName)
+        private ApplicationHostContext GetApplicationHostContext(Project project, FrameworkName frameworkName, IEnumerable<string> runtimeIdentifiers)
         {
-            var cacheKey = Tuple.Create("ApplicationContext", project.Name, frameworkName);
+            var cacheKey = Tuple.Create("ApplicationContext", project.Name, frameworkName, runtimeIdentifiers);
```

Comment:
It is a little gross because this method needs the cache context, but I think that's OK